tools/gocti_type_generator/pyproject.toml

```diff
@@ -13,7 +13,7 @@ license = {text  = "Apache 2.0"}
 version = "0.1.0"
```

Comment:
GoCTI version should be updated here as well.

Reply:
Thank you, I specified in the README that it should be updated here as well.

Reply:
Good point